docs/api/repl.md

```diff
@@ -0,0 +1,21 @@
+# REPL
+
+Read-Eval-Print-Loop (REPL) is a simple, interactive computer programming
+environment that takes single user inputs (i.e. single expressions), evaluates
+them, and returns the result to the user.
+
+The `repl` module provides a REPL implementation that can be accessed using:
+
+* Assuming you have `electron` or `electron-prebuilt` installed as a local
+  project dependency:
+
+  ```sh
+  ./node_modules/.bin/electron --interactive
+  ```
+* Assuming you have `electron` or `electron-prebuilt` installed globally:
+
+  ```sh
+  electron --interactive
```

Comment:
I think it would be good to mention this only creates a REPL for the main process. You can use the Console tab of the Dev Tools to get a REPL for the renderer processes.